rancher/distros-test-framework
# Loglevel env run

#### Proposed Changes ####

-  https://github.com/rancher/qa-tasks/issues/1232 




#### Types of Changes ####



#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

- WIP (  will link PR here ) 

2. Verify code lint; we should not have errors.
- YES ok

3. Update the documentation if needed.

- yes README
4. Update makefile and docker run if adding new tests.

- n/a
5. Run your tests at least 4 times with all configurations needed and possible.

- n/a
6. If needed test with different os types.

- n/a 

#### Linked Issues ####



#### Further Comments ####
